src/server/active_tran_server.cpp

```diff
@@ -183,16 +182,15 @@ active_tran_server::init_page_server_hosts (const char *db_name)
       const int local_exit_code = connect_to_page_server (node, db_name);
       if (local_exit_code == NO_ERROR)
 	{
-	  // found valid host clear the errors from the bad ones
 	  ++valid_connection_count;
 	  exit_code = NO_ERROR;
+	  //found valid host clear the errors rom the bad ones
```

Comment:
```suggestion
	  // found valid host; clear the error possibly remaining from previous failing ones
```
What if the last connection attempt is actually a failing one, but there is at least one valid one. Don't we want to `er_clear` in that case?

Reply:
moved er_clear so it is only called once